### ISSUE TITLE
Form input v9 focus indicator & WCAG 2.2

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#22970]
Our current focus indicator design on input-like form elements (so far, Input, Select, Textarea, Combobox, Spinbutton) does not meet the new WCAG 2.2 requirements for minimum focus indicator size: https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html

WCAG 2.2 is not yet a recommendation, but it would be good to make a plan for when it becomes one. The most important parts of the new requirement for us to pay attention to are:
1. the miniumum size of the focus indicator
2. The pixels that count towards the miniumum size need to have at least 3:1 contrast between the focused and unfocused state (so the pixels changing from a dark border to a blue focus line wouldn't count as part of the area).
